cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -2392,6 +2421,8 @@ func generateMountsModeRun(mst *initramfsMountsState) error {
 		}
 	}
 
+	diskState.serializeTo("unlocked.json")
```

Comment:
this need a step comment

Reply:
Not sure if you are if you are meaning step as the step numbers that are on some of the comments around. Are they reference to an external document? Because some step numbers are repeated. I am not totally sure why we have numbers here.